deploy.rb

```diff
@@ -10,7 +10,7 @@ def on_standby?
   met? {
     output = raw_shell('npm ls', :cd => path)
     # Older `npm` versions exit 0 on failure.
-    output.ok? && output.stdout['UNMET DEPENDENCY'].nil?
+    output.ok? && ['UNMET DEPENDENCY', '(empty)'].none? { |str| output.stdout[str] }
```

Comment:
@nickbrowne problem may be that `npm ls` won't show unmet development dependencies, and we have all our dependencies for analytics under `devDependencies`. So, even when there is an uninstalled dep, `npm ls` returns `0` and doesn't output `UNMET DEPENDENCY`. Should we move all the deps to `dependencies`?